test/expr/primary/literal/collection_upcast_opt.swift

```diff
@@ -0,0 +1,38 @@
+// RUN: %target-typecheck-verify-swift -dump-ast 2> %t.ast
+// RUN: %FileCheck %s < %t.ast
+
+// Verify that upcasts of array literals upcast the individual elements in place
+// rather than
```

Comment:
I think you forgot to commit the rest of the comment here :)

Reply:
Indeed. Fixed!